circle.yml

```diff
@@ -37,13 +40,24 @@ test:
   override:
     - docker run -v /var/run/docker.sock:/run/docker.sock -v /home/ubuntu:/home/go weaveworks/weave-build:
         parallel: true
-    - cd $SRCDIR/test; ./gce.sh setup:
-        parallel: true
-    - cd $SRCDIR/test; . ./gce.sh hosts; ./setup.sh:
-        parallel: true
-    - cd $SRCDIR/test; . ./gce.sh hosts; ./run_all.sh:
```

Comment:
One nice-ish side effect of having this as three commands is that you could see in the tests how long each one took.